kratos/includes/variables.h

```diff
@@ -369,6 +369,8 @@ namespace Kratos
 
     KRATOS_DEFINE_VARIABLE( Vector, ELEMENTAL_DISTANCES )
     KRATOS_DEFINE_VARIABLE( Vector, ELEMENTAL_EDGE_DISTANCES )
+    KRATOS_DEFINE_VARIABLE( Vector, ELEMENTAL_EXTRAPOLATED_EDGE_DISTANCES )
+    KRATOS_DEFINE_VARIABLE( Vector, ELEMENTAL_DISTANCES_WITH_EXTRAPOLATED )
```

Comment:
In order to be as consistent as possible with the already existent variables I suggest renaming `ELEMENTAL_EXTRAPOLATED_EDGE_DISTANCES` to `ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED`.

One thing that comes to my mind is why do we need a new variable for the `ELEMENTAL_DISTANCES_WITH_EXTRAPOLATED`? Couldn't we use the already existent `ELEMENTAL_DISTANCES`? (we could always check which edge has a extrapolated value with the `ELEMENTAL_EXTRAPOLATED_EDGE_DISTANCES` value).

Reply:
I think you are right, we could use the existing ELEMENTAL_DISTANCES. (I didn't wanna mess with the existing code too much.) Probably that would be more clear as well.

Reply:
As a consequence, we will have to check whether the element is incised before we check whether it is cut OR check whether it is incised in the IsCut() method.
What do you prefer?

Reply:
This has no technical implications right?

Reply:
After a brief discussion, it will have technical implications.
1. We need to check for the incised / not incised with the edge distances.
2. If not incised, we check for the completely intersected with the elemental distances.

(this have no implications in this PR but will be important once we modify the elements).